task5/src/utils/button/button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './button.scss';
+import PropTypes from 'prop-types';
+
+function Button({children, color, className, onClick}) {
+  const buttonStyle = `${(color || 'primary')} ${className || ''}`;
+  return (<button className={buttonStyle} onClick={onClick}>{children}</button>
+  );
+}
+
+Button.protoTypes = {
```

Comment:
I just noticed that you write `Button.protoTypes` and it is not protoTypes is propTypes so you will have to change all to `Button.propTypes`

https://reactjs.org/docs/typechecking-with-proptypes.html